app/graphql/types/account_theme.rb

```diff
@@ -0,0 +1,4 @@
+class Types::AccountTheme < Types::BaseEnum
+  value 'LIGHT', value: 'light'
+  value 'DARK', value: 'dark'
+end
```

Comment:
As of right now, we still haven't launched a dark theme.  We probably shouldn't add them in the GraphQL API until we know we're gonna use em